slices/libnode108.yaml

```diff
@@ -0,0 +1,23 @@
+package: libnode108
+
+slices:
+  libs:
+    essential:
+      # The chisel tool does not support per-architecture packages
+      # libatomic1 is only required on mips* and riscv64
+      # - libatomic1
```

Comment:
We will wait until that issue is resolved then apply the appropriate fix where needed. For now, we are going to keep that commented.

Reply:
Okay, in that case, let's add some comments/nudge in `nodejs.yaml` to install the `libatomic1_libs` additionally, since people will most likely be looking at that slice definition file.